src/users/users.service.ts

```diff
@@ -334,12 +339,20 @@ export class UsersService {
       data[data.length - 1].rank,
       1,
       countryCode,
-      eventType,
+      !!eventTypes?.length,
```

Comment:
Since this is used multiple places, storing a reference to it early in the function and re-using it makes sense?

```
const someEvents = !!eventTypes?.length
```

And then re-use it in 3 places?

Reply:
I'm guessing this  isbecause it's unclear what `!!eventTypes?.length` actually stands for, so I can factor it out to make that more clear into a descriptive variable.

Reply:
It should be more clear now.